.github/workflows/common-flow-container.yml

```diff
@@ -59,22 +68,44 @@ jobs:
         run: |
           export GITHUB_TOKEN="${{ secrets.GITHUB_TOKEN }}"
           ./deps/readies/bin/getredis --with-github-token
-      - name: Build
-        run: |
-          make SAN=${{ inputs.san }}
-          make COORD=1 SAN=${{ inputs.san }}
-      - name: Unit tests
-        run: |
-          make unit-tests LOG=1 CLEAR_LOGS=0 SAN=${{ inputs.san }}
-      - name: Flow tests
+
+      - name: Decide if to continue after failure
+        id: continue-on-error
         run: |
```

Comment:
Seems like this flow has a lot in common with common-flow-host... Consider making an even-more-common- flow.... Anyway, the same comments from common-flow-host applied here :)

Reply:
sadly it is immpossible, because GHA does not allow refering to `container:` without supplying a container, so I had to duplicate this file, with specific differences in the beggining of the file.
As an alternative (until GHA will support not refering to a container and behaving as running directly on the runner), I made `check-common-flow-sync.yml` flow, that verify that the logic part of both files are in sync, making it immposible to edit one and not theother, without failing the CI.